### ISSUE TITLE
v5.0.3 is incompatible with webpack 4

### ISSUE THREAD

**Issue by davidwebster48** [DylanPiercey/spawn-server-webpack-plugin#14]
I'm using webpack 4.44.2 and I just tried updating spawn-server-webpack-plugin from version 5.0.2 to 5.0.3.

I got an error when starting my server:

```
.../node_modules/spawn-server-webpack-plugin/dist/index.js:160
        result[existsAt] = fs.readFileSync(existsAt, "utf-8");
                              ^

TypeError: fs.readFileSync is not a function
```

My understanding is that you should increment your major version number when making a change that's not backwards compatible.

I'll stick with version 5.0.2 for now.

**Comment by DylanPiercey** [comment]
Ah, that's my bad. I thought it was still compatible. I'll get out a release that supports both. Thanks for creating the issue.

**Comment by DylanPiercey** [comment]
Should be fixed in `5.0.4`
